projects/ccd-case-ui-toolkit/src/lib/shared/components/search-result/search-result.component.html

```diff
@@ -30,6 +30,7 @@ <h2 class="heading-h2" id="search-result-heading__text" tabindex="-1">{{ (caseSt
         <div class="govuk-checkboxes__item">
           <input class="govuk-checkboxes__input" id="select-all" name="select-all" type="checkbox" (change)="selectAll()" [checked]="allOnPageSelected()" [disabled]="!canAnyBeShared()" />
           <label class="govuk-label govuk-checkboxes__label" for="select-all">
+            Select All
```

Comment:
This should be "Select all" - style guide is to use Sentence case, not Title Case